src/main/java/seedu/dengue/logic/commands/ClearCommand.java

```diff
@@ -2,22 +2,29 @@
 
 import static java.util.Objects.requireNonNull;
 
-import seedu.dengue.model.DengueHotspotTracker;
+import java.util.ArrayList;
+import java.util.List;
+
 import seedu.dengue.model.Model;
+import seedu.dengue.model.person.Person;
 
 /**
- * Clears the Dengue Hotspot Tracker.
+ * Clears all persons in the current displayed list of the Dengue Hotspot Tracker.
  */
 public class ClearCommand extends Command {
-
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Dengue Hotspot Tracker has been cleared!";
+    public static final String MESSAGE_SUCCESS = "%1$s persons cleared!";
```

Comment:
might be better if there can be a way such that if only one person is cleared, the message changes from plural persons to only person?